### ISSUE TITLE
Leading 0 from zip code is not included/stripped

### ISSUE THREAD

**Issue by sbernheim4** [bluesmoon/node-geoip#132]
I am in a zip code that begins with a 0 but the zip code I get back from using this does not include the 0. For example, if the zip code is 02345 then I only get back 2345 with the first 0 missing.

**Comment by bluesmoon** [comment]
Yeah, this is a known issue. See https://github.com/bluesmoon/node-geoip/pull/122#issuecomment-288784712

Do you want to take a stab at fixing it?

**Comment by sbernheim4** [comment]
Yeah I can try, should I just fork it and then submit a PR if I can get it to work?

**Comment by bluesmoon** [comment]
yes, that would be awesome.

**Comment by hotdang-ca** [comment]
Please review above PR.
My only concern is the return of a string, rather than a number; if backwards-compatibility requires zip to be a number, I could provide an optional property to grab `zipAsString`, or something...

**Comment by Download** [comment]
Zip codes as Number?

My zip code (Netherlands):

`6512 BM`

How will you turn that into Number? Or is this a code branch specifically for US zip codes?

**Comment by bluesmoon** [comment]
Maxmind only supports zip for the US.

It's still a big to use a number instead of string but it's hard to change.
